Makefile

```diff
@@ -0,0 +1,13 @@
+all: get build
+
```

Comment:
since you're adding a makefile, would you add a `release:` section that does:
```
 CGO_ENABLED=0 GOARCH=amd64 go build -o vcfanno_linux64 --ldflags '-extldflags "-static"' vcfanno.go
 GOOS=darwin GOARCH=amd64 CGO_ENABLED=0 go build -o vcfanno_osx --ldflags '-extldflags "-static"' vcfanno.go
```

Reply:
Since you already know what you want in the makefile, can you do it?